.docker/.volumes

```diff
@@ -0,0 +1,3 @@
+volumes=()
```

Comment:
Should this have a shebang or something to make it a bit more obvious it's Bash?

Reply:
Should not be executable in isolation, but only sourced, I was thinking of a better extension

Reply:
Perhaps leave a comment at the top of the file explaining the intended use case.